hikari/stickers.py

```diff
@@ -157,7 +157,7 @@ def image_url(self) -> files.URL:
         """
         ext = "json" if self.format_type is StickerFormatType.LOTTIE else "png"
 
-        return routes.CDN_STICKER.compile_to_file(urls.CDN_URL, hash=self.id, file_format=ext)
+        return routes.CDN_STICKER.compile_to_file(urls.CDN_URL, hash=str(self.id), file_format=ext)
```

Comment:
the field should be renamed to sticker_id instead of casting to str here

Reply:
Changed :+1: